Melanzana.MachO.Tests/RoundtripTests.cs

```diff
@@ -1,12 +1,16 @@
-using Xunit;
 using System.IO;
+using System.IO.Compression;
 using System.Linq;
-using Melanzana.Streams;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
 
 namespace Melanzana.MachO.Tests
 {
     public class RoundtripTests
     {
+        private const string WebDriverAgentFileName = "WebDriverAgentRunner-Runner.zip";

```

Comment:
Do we already have this in .gitignore? If not, should we add it?

Reply:
In my setup, this asset gets saved to the working directory of the unit tests.  That's the output folder (`bin/Debug`) or similar.  So it gets covered by that line in `.gitignore`.

Are there scenarios in which the working directory of the tests would not be the output directory?

Reply:
Just wanted to make sure that's the case. I don't think there's currently any scenario where a different output path is used. Thanks for confirmation!